charts/vault-monitoring/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
Should we make this optional? (ie. with `{{ if .Values.value.serviceMonitor.create }}` or similar).

That way this could also be deployed on a cluster that uses other discovery mechanisms in prom.

Reply:
you're right

Reply:
* [ ] TODO